File: Simulation/TrackerRunner/International/Vehicle.cs
using System.Text.Json.Serialization;

namespace TrackerRunner.International;

public class Vehicle
{
    public Vehicle(string id, string vehicleClassification, string fuelType)
    {
        Id = id;
        VehicleClassification = vehicleClassification;
        FuelType = fuelType;
    }

    [JsonPropertyName("id")]
    public string Id { get; set; }
    [JsonPropertyName("vehicleClassification")]
    public string VehicleClassification { get; set; }
    [JsonPropertyName("fuelType")]
    public string FuelType { get; set; }
}

File: Router-Service/src/router_service/models/vehicle.py
"""
The vehicle model.
"""
from pydantic import BaseModel


class VehicleInt(BaseModel):
    """
    Pydantic vehicle model as agreed with international teams.
    """

    id: str
    vehicleClassification: str
    fuelType: str


class Vehicle(VehicleInt):
    """
    Pydantic vehicle model.
    """

    licence: str
